src/hooks/useTestSettings.ts

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import { Element } from "@/Element";
+import { useSettings, Settings, ElementsSettings } from "./useSettings";
+import { useElements } from "./useElements";
+
+export function useTestSettings(
```

Comment:
Future tests won't use the same logic to store their settings, so we should mark this as legacy and deprecate its use. I'd call it `useLegacyTestSettings` and mark it as deprecated.

Reply:
Sounds like a feature/enhancement, not really within the scope of refactor to Hooks